modules/compiler/vecz/source/transform/control_flow_conversion_pass.cpp

```diff
@@ -1356,6 +1372,45 @@ bool ControlFlowConversionState::Impl::applyMaskToCall(CallInst *CI,
   return true;
 }
 
+bool ControlFlowConversionState::Impl::applyMaskToAtomicRMW(
```

Comment:
This function has a single `return` statement, `return true;`. There is no path where it returns that we cannot vectorize. Does this mean we can just return `void` and avoid having to deal with the possibility of failure in the caller, or is this anticipating future work?

Reply:
Oh, hidden in the `VECZ_FAIL_IF` macro. Never mind.